metafacture-flux/src/main/java/org/metafacture/flux/HelpPrinter.java

```diff
@@ -113,6 +123,32 @@ private static <T> void describe(final String name, final ObjectFactory<T> facto
 
         final Map<String, Method> attributes = configurableClass.getSetters();
 
+        printAttributes(out, configurableClass, attributes);
+
+        String inString = "<unknown>";
+        String outString = "";
+        final In inClass = moduleClass.getAnnotation(In.class);
+        if (inClass != null) {
+            inString = inClass.value().getSimpleName();
+        }
+        final Out outClass = moduleClass.getAnnotation(Out.class);
+        if (outClass != null) {
+            outString = outClass.value().getSimpleName();
+        }
+        out.println("- signature:\t" + inString + " -> " + outString);
+        if (examplesMap.get(name) != null && examplesMap.get(name).size() == COLUMN_TO_EXAMPLE) {
+            out.println("- [example in Playground]" + "(" + examplesMap.get(name).get(2) + ")");
```

Comment:
Reuse the constant (and variable)?

```suggestion
            out.println("- [example in Playground]" + "(" + commandInfo.get(COLUMN_TO_EXAMPLE - 1) + ")");
```